### PR TITLE
link and headers fix

### PR DESCRIPTION
**Pull request by josanri**
Due to missing slash '/' when traversing through sponsor tab to schedule, the page got stuck in sponsor-page#schedule, which does not exist.

Also changed html tag for headers  to improve screen readers heading order.

### CODE REVIEW

File: src/components/Speaker.astro [hunk 1]
```diff
@@ -31,9 +31,9 @@ const { speaker, index } = Astro.props;
   <div
     class="flex flex-col align-middle justify-center gap-1 md:gap-3 py-[0px] text-center lg:text-left"
   >
-    <h3 transition:name={`name-speaker-${index}`} class="text-2xl font-bold">
```
Comment (davidrojom) [inline src/components/Speaker.astro]: this is supposed to be a heading, is there any reason for this change? 
image

File: src/components/TeamMember.astro [hunk 2]
```diff
@@ -27,10 +27,10 @@ const { member, index, kind } = Astro.props;
       width={200}
     />
   </a>
-  <h4
```
Comment (davidrojom) [inline src/components/TeamMember.astro:30]: same as above

File: src/components/Navigation.astro [hunk 3]
```diff
@@ -40,7 +40,7 @@ const featuredSponsors = DATA.sponsors.filter(
           <li class="mx-auto">
             <a
               aria-label="Go to Schedule"
-              href="#schedule"
```
Comment (davidrojom) [inline src/components/Navigation.astro:43]: good catch

File: src/components/Footer.astro [hunk 4]
```diff
@@ -42,9 +42,9 @@ import Socials from "./Socials.astro";
     {
       !!DATA.previousEditions.length && (
         <div class="flex flex-col gap-6 w-fit mx-auto">
-          <h5 class="text-center lg:text-left font-semibold text-lg">
```
Comment (davidrojom) [inline src/components/Footer.astro:45]: this is not supposed to be an h2, due to footer section having a lower level of impact than main content sections 
image
image

**Review by davidrojom — CHANGES_REQUESTED** [review]

File: src/components/Speaker.astro (hunk 1 above)
Comment (josanri) [reply src/components/Speaker.astro]: Although it is a strong text, I'm not sure if it should be a heading as it makes the web headings map a bit confusing or hard to traverse with each person count as a heading:
image

File: src/components/TeamMember.astro (hunk 2 above)
Comment (josanri) [reply src/components/TeamMember.astro:30]: Same [reason as above](https://github.com/biznaga-fest/web/pull/8#discussion_r1769565267), but here there was also a heading jump from h2 to h4:

File: src/components/Footer.astro (hunk 4 above)
Comment (josanri) [reply src/components/Footer.astro:45]: By keeping it as `<h5>`, the heading is semantically like a part of the latest section:
image

File: src/components/Speaker.astro (hunk 1 above)
Comment (davidrojom) [reply src/components/Speaker.astro]: It's not a paragraph, though

File: src/components/TeamMember.astro (hunk 2 above)
Comment (davidrojom) [reply src/components/TeamMember.astro:30]: Its not a paragraph, though. Use h3 instead

Comment (josanri) [reply src/components/TeamMember.astro:30]: Done at [b0df69b0ec652dd5f866913fc00ffc4144cddebc ](https://github.com/biznaga-fest/web/pull/8/commits/b0df69b0ec652dd5f866913fc00ffc4144cddebc)

File: src/components/Speaker.astro (hunk 1 above)
Comment (josanri) [reply src/components/Speaker.astro]: Left as heading name at [b0df69b0ec652dd5f866913fc00ffc4144cddebc](https://github.com/biznaga-fest/web/pull/8/commits/b0df69b0ec652dd5f866913fc00ffc4144cddebc)

**Comment by davidrojom** [comment]
lgtm

**Pull request merged**